tests/rptest/tests/shadow_indexing_tx_test.py

```diff
@@ -76,6 +56,10 @@ def setUp(self):
             rpk.alter_topic_config(topic.name, 'redpanda.remote.write', 'true')
             rpk.alter_topic_config(topic.name, 'redpanda.remote.read', 'true')
 
+    def tearDown(self):
+        self.s3_client.empty_bucket(self.s3_bucket_name)
```

Comment:
Isn't empty_bucket called automatically in RedpandaService?

Reply:
You're right, I was fixated on the setup/teardown path and ignoring the `clean()` hook on services.